compiler/packages/babel-plugin-react-compiler/src/HIR/Environment.ts

```diff
@@ -237,8 +242,10 @@ const EnvironmentConfigSchema = z.object({
    * Enables inlining ReactElement object literals in place of JSX
    * An alternative to the standard JSX transform which replaces JSX with React's jsxProd() runtime
    * Currently a prod-only optimization, requiring Fast JSX dependencies
+   *
+   * The symbol configuration is set for backwards compatability with pre-React 19 transforms
    */
-  enableInlineJsxTransform: z.boolean().default(false),
+  inlineJsxTransform: ReactElementSymbolSchema.nullish(),
```

Comment:
this format of a nullable string makes it harder or uglier to add more options in the future w/o breaking changes. You could consider a nullable object with a `elementSymbol` key.

If it's only for internal experimentation, it doesn't really matter that much.